packages/orbit-components/src/Box/Box.stories.js

```diff
@@ -0,0 +1,272 @@
+// @flow
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { text, select, object } from "@storybook/addon-knobs";
+
+import Text from "../Text";
+import RenderInRtl from "../utils/rtl/RenderInRtl";
+import {
+  DISPLAY,
+  WRAP,
+  SHRINK,
+  GROW,
+  OVERFLOW,
+  COLORS,
+  DIRECTION,
+  ALIGN,
+  SPACINGS,
+  TEXT_ALIGN,
+  JUSTIFY,
+  WIDTH_AND_HEIGHT,
+  ELEVATION,
+  BORDER_RADIUS,
+  POSITION,
+} from "./consts";
+
+import Box from "./index";
+
+const DEFAULT_COLOR = "blueDark";
+
+storiesOf("Box", module)
+  .add(
+    "Default",
+    () => {
+      const children = text("Children", "Default box");
+      const as = text("As", "span");
+      return (
+        <Box as={as}>
+          <Text>{children}</Text>
+        </Box>
+      );
+    },
+    {
+      info:
+        "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+    },
+  )
+  .add("Overflow", () => {
+    const overflow = select("overflow", OVERFLOW, OVERFLOW.AUTO);
```

Comment:
Since we have changed the definition of consts from arrays to objects, we should also change it in the stories.
e.g. `const overflow = select("overflow", Object.keys(OVERFLOW), OVERFLOW.AUTO);`

I noticed that in some cases the `OVERFLOW[0]` is used and that's also incorrect.